server/middlewares/controllersValidator/ReqBodyValidator.js

```diff
@@ -0,0 +1,70 @@
+import Validator from '../../helpers/Validator';
+
+class ReqBodyValidator {
+  static validateFood(req, res, next) {
+    const {
+      name,
+      description,
+      price,
+      imageurl
+    } = req.body;
+
+    if (!name || !description || !price || !imageurl) {
+      return res.status(400).json({
+        message: 'Bad request, no field should not be missing',
+      });
+    }
+
+    if (!Validator.hasAminLength(name) || !Validator.hasAminLength(description)) {
+      return res.status(400).json({
+        message: 'Fields length must not be less than three',
```

Comment:
Don't you think this is a little bit more descriptive?  
**message: 'Fields length must not be less than three characters',**

Reply:
Just seeing this. Network was slow. Let me implement it